pcsx2/ps2/Iop/IopHwRead.cpp

```diff
@@ -36,7 +39,15 @@ mem8_t __fastcall iopHwRead8_Page1( u32 addr )
 	mem8_t ret;		// using a return var can be helpful in debugging.
 	switch( masked_addr )
 	{
-		mcase(HW_SIO_DATA): ret = sioRead8(); break;
+		mcase(HW_SIO_DATA) :
+			ret = sioRead8();
+			ret |= sioRead8() << 8;
+			if ( 1 /*sizeof(T) == 4*/)
```

Comment:
"T" isn't declared at that point. What should the correct handling be here?
I quickfixed it by choosing "ret |= sioRead8() << 16;" via coin flip.